packages/root-cms/core/app.tsx

```diff
@@ -277,7 +278,13 @@ function getRefererOrigin(req: Request): string {
   return refererOrigin;
 }
 
-function cachebust(url: string) {
-  const ts = Math.floor(new Date().getTime() / 1000);
-  return `${url}?c=${ts}`;
+/** Modify the given URL to bust the cache. */
+function cachebust(req: Request, url: string) {
+  // In local dev, use the the timestamp to cachebust.
+  // In non-local mode, use the package version.
+  const value =
+    req.hostname === 'localhost'
```

Comment:
can you confirm if hostname would ever include the port?

Reply:
Yeah; I tried this and it worked locally, so it should be good.